doc/app/assets/styles/style.scss

```diff
@@ -3,6 +3,7 @@
 
 @import '~milligram/src/milligram';
 
+@import './admonitions';
```

Comment:
Hidden feature 👀

Reply:
Not really, it's documented already 😄  https://stenopephp.github.io/Stenope/supported-formats/#admonitions